src/components/Search/SearchInput/SearchInput.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './SearchInput.css';
+
+interface SearchInputProp {
+  searchQuery: string;
+  setSearchQuery: (value: string) => void;
+}
+
+function SearchInput(props: SearchInputProp) {
+  const { searchQuery, setSearchQuery } = props;
+  const handleChange = (event: React.SyntheticEvent) => {
```

Comment:
SyntheticEvent is a general type, be more specific and you use React.ChangeEvent<>